File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {
	public List<TbBrand> findAll();
	/**
	 * 品牌分页
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	/**
	 * 新增
	 * @param brand
	 */
	public void add(TbBrand brand);
	/**
	 * 根据ID查找
	 * @param id
	 * @return
	 */
	public TbBrand findOne(Long id);
	/**
	 * 修改
	 * @param brand
	 */
	public void update(TbBrand brand);
	/**
	 * 删除
	 * @param id
	 */
	public void delete(Long[] id);
	
	public PageResult findPage(TbBrand brand,int pageNum,int pageSize);
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
//品牌服务层
     app.service("brandService",function($http){
    	 this.findAll=function(){
    		 return $http.get("../brand/findAll.do");
    	 }    	 
    	 this.findPage=function(page,size){
    		 return $http.get("../brand/findPage.do?page="+page+"&size="+size);
    	 }
    	 this.findOne=function(id){
    		 return $http.get("../brand/findOne.do?id="+id);
    	 }
    	 this.dele=function(ids){
    		 return $http.get("../brand/delete.do?ids="+ids);
    	 }
    	 this.search=function(page,size,searchEntity){
    		 return $http.post("../brand/search.do?page="+page+"&size="+size,searchEntity);
    	 }
    	 this.update=function(entity){
    		 return $http.post("../brand/save.do",entity);
    	 }
    	 this.add=function(entity){
    		 return $http.post("../brand/add.do",entity);
    	 }
    	 this.selectOptionList=function(){
    		 return $http.get("../brand/selectOptionList.do");
    	 }
     });